apps/test/unit/lib/levelbuilder/script-editor/UnitEditorTest.jsx

```diff
@@ -427,7 +427,7 @@ describe('UnitEditor', () => {
 
     it('shows error when version year is set but family name is not', () => {
       sinon.stub($, 'ajax');
-      const wrapper = createWrapper({});
+      const wrapper = createWrapper({initialIsCourse: true});
```

Comment:
Maybe in a follow up, but we should add a test for when initialIsCourse is false.

Reply:
I thought about that. But the editor does not allow setting the fields unless you check is course first so it seemed like a case that could not really happen. Happy to add the test tho if we think its still worth it?

Reply:
Yeah I guess I would create a test for exactly the situation we're seeing here (family name or version year set but not both) and check that save works for units in courses. It's not a situation we should get into but we obviously have

Reply:
Added